src/JanusFtl.cpp

```diff
@@ -410,7 +423,7 @@ void JanusFtl::initVideoDecoders()
 
 void JanusFtl::initOrchestratorConnection()
 {
-    if (configuration->GetNodeKind() != NodeKind::Standalone)
+    if (!empty(configuration->GetOrchestratorHostname()))
```

Comment:
Is this `std::empty` ?

Prefer qualifying the `std` namespace or `!(configuration->GetOrchestratorHostname().empty())`

Reply:
Thanks, just throwing stickers at the wall here :D